src/components/Dropdown.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+interface Props {
+  people: Person[],
+  query: string,
+  selectedUser: Person | null,
+  setQuery: (newValue: string) => void,
+  setSelectedUser: (newPerson: Person) => void
+  handleQueryChange: (personName: string) => void
+  appliedQuery: string
+  setAppliedQuery: (personName: string) => void
+}
+
+export const Dropdown: React.FC<Props> = ({
+  people,
+  query,
+  selectedUser,
+  setQuery,
+  setSelectedUser,
+  handleQueryChange,
+  appliedQuery,
+  setAppliedQuery,
+}) => {
+  const handleChoseUser = (person: Person) => {
+    setSelectedUser(person);
+    setAppliedQuery(person.name);
+    setQuery(person.name);
+  };
+
+  return (
+    <div className={classNames('dropdown', {
+      'is-active': selectedUser?.name !== appliedQuery,
+    })}
+    >
+      <div className="dropdown-trigger">
+        <input
+          type="text"
+          placeholder="Enter a part of the name"
+          className="input"
+          value={query}
+          onChange={(event) => handleQueryChange(event.target.value)}
+        />
+      </div>
+
+      {appliedQuery && (
```

Comment:
according to readme : hide suggestions on text change. You can add a check if appliedQuery === query to avoid displaying a list of people while typing